package.json

```diff
@@ -16,6 +16,7 @@
   ],
   "activationEvents": [
     "onCommand:extension.runCode",
+    "onCommand:extension.stopAll",
```

Comment:
Should we change here to `onCommand:extension.restart` ?

Reply:
yes, of coourse! (fixed)